src/behavior/src/A32NX_Interior_FCU.xml

```diff
@@ -288,19 +288,23 @@
                     } els{
                         (&gt;H:A320_Neo_FCU_VS_INC_VS)
                     }
+                    (&gt;H:A320_Neo_CDU_VS)
```

Comment:
As mentioned above, maybe those events can be skipped.

Reply:
I'd rather use events over useless x times per seconds checks. Also this allows for consistency.